workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -217,6 +217,22 @@ def _get_metric(self, metric_type: str, variable: str, run: str) -> pd.Series:
         return _metrics[_metrics.run == run]
 
 
+@dataclass
+class RunMovieUrls:
```

Comment:
The definition of `movie_name = os.path.basename(url)` is important too—I guess it could go into the calling context but it is appropriate to be here IMO.